packages/kununu-utils/kununu-logger/index.js

```diff
@@ -1,58 +1,76 @@
 import {createLogger, transports, format} from 'winston';
 
+const stringify = require('json-stringify-safe');
+
 const {timestamp, printf} = format;
 const getColorizedMessage = message => `\x1b[32m${message}\x1b[0m`;
 
 const minimumLogLevel = process.env.MINIMUM_LOG_LEVEL || 'info';
 
-/**
- * Format request and response data
- * @param  object containing res and req
- * @return string stringified object
- */
-export const formatNodeRequest = ({
-  req,
-  res,
-  label,
-  timeTakenMicros,
-}) => JSON.stringify({
-  label,
-  time: new Date().toISOString(),
-  method: req.method,
-  request: req.originalUrl,
-  status: res.statusCode,
-  remote_ip: req.connection.remoteAddress || '-',
-  referer: req.headers.referer || '-',
-  forwarded_for: req.headers['x-forwarded-for'] || '-',
-  trace_id: req.headers['x-amzn-trace-id'] || '-',
-  logType: 'middleware_logger',
-  user_agent: req.headers['user-agent'] || '-',
-  time_taken_micros: timeTakenMicros,
-  build: process.env.BUILD_NAME || '-',
-});
+export const logLevelNum = {
+  emergency: 0,
```

Comment:
Our log levels that are also logging methods to call, e.g.: `logger.emergency({...});`